README.md

```diff
@@ -95,37 +95,12 @@ https://github.com/DeloitteDigitalAT/core-banking/blob/master/devops/config/chec
 
 Note: make sure that the required softwares are installed on the system (see section above)
 
-**1. Build containers**
-
-```
-cd data-access-services
-./docker-build.sh
-
-cd ../domain-service/credit-application/
-./docker-build.sh
-
-cd ../data-service
-./docker-build.sh
-```
-
-**2. Run docker compose**
-
-From root directory:
-```
-cd devops/docker/
-docker-compose up -d
-```
-**3. Check processes to verify**
+Run the bootstrap script from root directory:
 ```
-docker ps
+./core_banking_bootstrap.sh
```

Comment:
Could we pls use dash instead of underscore in sh script names? Existing ones like `docker-build` is already written like that, it would be better to make them consistent.